src/Database/Driver/Mysql.php

```diff
@@ -27,10 +26,12 @@
 
 /**
  * Class Mysql
+ *
+ * @internal
```

Comment:
Do we really need to mark the driver classes as internal?

Reply:
I leave it up to you.  Are they supportable classes?

We want people to extend `Driver` and implement `DriverInterface`. We want people to extend `SchemaDialect`, but do we want anything else?

Reply:
I have a feeling someone might want to extend say the Mysql driver and override something to suite their special needs.

Reply:
I can revert the @internal flag. Just means we can't change any of the protected implementation functions.

Reply:
As per our BC guidelines protected methods can be changed in new minor release.